packages/node_modules/@ciscospark/internal-plugin-metrics/src/config.js

```diff
@@ -8,7 +8,8 @@ import {inBrowser} from '@ciscospark/common';
 export default {
   device: {
     preDiscoveryServices: {
-      metricsServiceUrl: process.env.METRICS_SERVICE_URL || `https://metrics-a.wbx2.com/metrics/api/v1`
+      metricsServiceUrl: process.env.METRICS_SERVICE_URL || `https://metrics-a.wbx2.com/metrics/api/v1`,
+      diagnosticsServiceUrl: process.env.DIAGNOSTICS_SERVICE_URL || `https://metrics-a.wbx2.com/metrics/api/v1`
```

Comment:
Right now the call diagnostic event API is the same as clientmetrics API. However, from Feb to now, the API has been changed three times. I am not sure whether they will change it some day. This is the reason I want to use a variable to present the `diagnosticsServiceUrl` with a default value. 

It is not necessary to put it before the user logs in. But I am not sure where I should put it.